privacy-config/privacy-config-impl/src/main/java/com/duckduckgo/privacy/config/impl/PrivacyConfigPersister.kt

```diff
@@ -115,7 +115,7 @@ class RealPrivacyConfigPersister @Inject constructor(
                 // Then feature flags
                 jsonPrivacyConfig.features.forEach { feature ->
                     feature.value?.let { jsonObject ->
-                        privacyFeaturePluginPoint.getPlugins().firstOrNull { feature.key == it.featureName }?.let { featurePlugin ->
+                        for (featurePlugin in privacyFeaturePluginPoint.getPlugins().filter { feature.key == it.featureName }) {
```

Comment:
This change is needed because in code generation we can't modify classes, just add more.
The remote feature flags for the plugin point and its plugins are effectively different classes related to the same feature. Example:
The plugin point remote feature would be
```kotlin
@ContributesRemoteFeature(
  scope = AppScope::class,
  featureName = "pluginPointMyPlugin",
)
public interface MyPluginPoint_RemoteFeature {
  @Toggle.DefaultValue(defaultValue = true)
  public fun self(): Toggle
}
```
while the remote feature associated to one of the plugins would be
```kotlin
@ContributesRemoteFeature(
  scope = AppScope::class,
  featureName = "pluginPointMyPlugin",
)
public interface FooMyPlugin_RemoteFeature {
  @Toggle.DefaultValue(defaultValue = true)
  public fun self(): Toggle

  @Toggle.DefaultValue(defaultValue = true)
  public fun pluginFooMyPlugin(): Toggle
}
```

The both refer to the same remote config feature `pluginPointMyPlugin` and both need to be called by the persister for all to work correctly